src/platforms/php/common/performance.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Performance
+description: "Learn more about how to configure our Performance integrations to get the best experience out of it."
+---
+
```

Comment:
We could use an introductory paragraph to break the ice and set the context (good for SEO and for readers) before diving into specific config.